source/robot_model/include/robot_model/Model.hpp

```diff
@@ -234,6 +234,12 @@ class Model {
    */
   void set_gravity_vector(const Eigen::Vector3d& gravity);
 
+  /**
+   * @brief Getter of the pinocchio model
+   * @return the pinocchio model
+   */
+  pinocchio::Model get_pinocchio_model() const;
```

Comment:
Do we actually want a copy there or a const reference? I am tempted to think the later but maybe a copy is fine.

Reply:
well returning the const reference was my original suggestion to patrick in person, which is useful when using as a const rvalue (especially in the context of exposing internal getters such as `robot_model.get_pinocchio_model().lowerPositionLimit`). It just prevents you from doing anything with it as an lvalue. 

I thought a normal copy (return value) would be fine as well since we probably won't use it much, so efficiency is not a concern. But actually in retrospect the example above (using it just to access lower-level const functions) would be the main usage for now and make more sense as a const reference. In addition, it's unlikely that we would actually want to use it in any way that a const reference would prohibit.

Reply:
> > Actually you would even be able to do something like `const pinocchio::Model& model = robot_model.get_model()` if you want to store the model for a later usage.
> 
> I tried but I get the same error, something like this:
> 
> ```
> /usr/include/boost/variant/variant.hpp:2347:41: error: using invalid field ‘boost::variant<T0, TN>::storage_’
>  2347 |               which_, which(), visitor, storage_.address()
> ```

I get this error even if I make `robot_model_` public.